README.md

```diff
@@ -1,3 +1,8 @@
+[Important] Deprecated Branch
+======================================================================================================================================================
+[Master branch](https://github.com/gengo/gengo-python) is supporting both Python 2 and 3. No longer you don't need to use this branch for Python 3.
+We are going to delete this branch after the end of July 2017.
```

Comment:
`No longer you don't need to use this branch for Python 3.` :point_right: `You no longer need to use this branch for Python 3.`